functions/user.js

```diff
@@ -1,80 +1,74 @@
-const config = require("./config");
+const _ = require("lodash");
 const GitHubbub = require('./githubbub');
-const util = require("util");
 const Promise = require('bluebird');
-const _ = require("lodash");
+const ServerValue = require('firebase-admin').database.ServerValue;
```

Comment:
might be a bit more clear to do something like this: 
```const SERVER_TIMESTAMP = require('firebase-admin').database.ServerValue.TIMESTAMP;```

or... looks like every time we need ServerValue.TIMESTAMP, we also have db, so maybe access right from there?